src/telliot_feeds/reporters/flashbot.py

```diff
@@ -104,38 +98,77 @@ async def report_once(
             _nonce=timestamp_count,
             _queryData=query_data,
         )
+        # Estimate gas usage amount
+        gas_limit, status = self.submit_val_tx_gas_limit(submit_val_tx=submit_val_tx)
+        if not status.ok or gas_limit is None:
+            return None, status
+
+        self.gas_info["gas_limit"] = gas_limit
+        # Get account nonce
         acc_nonce, nonce_status = self.get_acct_nonce()
         if not nonce_status.ok:
             return None, nonce_status
 
         # Add transaction type 2 (EIP-1559) data
         if self.transaction_type == 2:
-            logger.info(f"maxFeePerGas: {self.max_fee}")
-            logger.info(f"maxPriorityFeePerGas: {self.priority_fee}")
+            if not self.max_fee:
```

Comment:
I'm gonna leave this for now. I think we should consolidate the reporter class since the old oracle contracts are deprecated or refactor the main class to be geared to the newest oracle.